python/mooseutils/VectorPostprocessorReader.py

```diff
@@ -115,7 +115,6 @@ def update(self, time=None):
 
         # The list of files from the supplied pattern
         last_modified = 0.0
-        self._frames = dict()
```

Comment:
@bwspenc The VPP reader was reading all the files again with every call to update because of this little guy